Shared/Action/TelemetryAction.swift

```diff
@@ -126,14 +126,12 @@ class TelemetryActionHandler: ActionHandler {
 
     lazy var telemetryActionListener: AnyObserver<TelemetryAction> = {
         return Binder(self) { target, action in
-            let extras = self.addUidTo(extras: action.extras)
-
             target.telemetry.recordEvent(
                     category: TelemetryEventCategory.action.rawValue,
                     method: action.eventMethod.rawValue,
                     object: action.eventObject.rawValue,
                     value: action.value,
-                    extras: extras
+                    extras: nil
```

Comment:
This removes all extras from the event; is this what you intend?

Also: can we remove `addUidTo()` method?